dist/jquery.interactive-world-map-country-popup.coffee

```diff
@@ -0,0 +1,58 @@
+$ ($, window, document) ->
+  pluginName = 'interactiveWorldMapCountryPopup'
+  settings   = null
+  element    = null
+  mainModal  = null
+
+  defaults =
+    popup:
+      margins:
+        top:  100
+        left: 60
+
+  Popup = (options) ->
+    @_options = options;
+    @init();
+
+  $.extend Popup.prototype,
+    init: ->
+      settings  = $.extend(defaults, @_options)
+      element   = $('g', settings.element).find("[data-code='" + settings.code  + "']");
+      mainModal = element.parents(':eq(4)');
+
+  $.fn[pluginName] = (options) ->
+
+    jQuery(this).each ->
+      new Popup (options)
+      $offset = element.offset()
+      $modal  = mainModal.find('.' + settings.classes.modal)
+      $modal.toggleClass('active')
+
+      # Calculate the distance between the country and the top of the browser for show with the correctly space the popup
+      distanceXtop = $offset.top  - settings.margins.top
+      distanceLeft = $offset.left + settings.margins.left
+      countryName  = settings.country.name
+      websites     = settings.country.websites
+      $('.' + settings.classes.containerWebsites).empty()
+
+
+      # If has something appended inside from another country
+      $.each websites, (index, web) ->
+        classBtn = if web.main then settings.classes.websites else 'regularBtn'
```

Comment:
Hmmm... again matter of taste... I prefer `? :` operator. But as you like it :smile: